models/rnn.py

```diff
@@ -54,18 +54,7 @@ def prep_model(model, N, s0pad, s1pad, c):
                 dropout=c['dropout'], dropoutfix_inp=c['dropoutfix_inp'], dropoutfix_rec=c['dropoutfix_rec'],
                 sdim=c['sdim'],
                 rnnbidi=c['rnnbidi'], rnn=c['rnn'], rnnact=c['rnnact'], rnninit=c['rnninit'],
-                rnnbidi_mode=c['rnnbidi_mode'], rnnlevels=c['rnnlevels'])
+                rnnbidi_mode=c['rnnbidi_mode'], rnnlevels=c['rnnlevels'],return_sequences=True)
 
-    # Projection
-    if c['project']:
-        model.add_shared_node(name='proj', inputs=['e0s_', 'e1s_'], outputs=['e0p', 'e1p'],
-                              layer=Dense(input_dim=int(N*c['sdim']), output_dim=int(N*c['pdim']),
-                                          W_regularizer=l2(c['l2reg']), activation=c['pact']))
-        # This dropout is controversial; it might be harmful to apply,
-        # or at least isn't a clear win.
-        # model.add_shared_node(name='projdrop', inputs=['e0p', 'e1p'], outputs=['e0p_', 'e1p_'],
-        #                       layer=Dropout(c['dropout'], input_shape=(N,)))
-        # return ('e0p_', 'e1p_')
-        return ('e0p', 'e1p')
-    else:
-        return ('e0s_', 'e1s_')
+    # Projection deleted
```

Comment:
I guess we shouldn't merge this, right? :) Can't we just set project=False if it's not appropriate for some model or task/model combination?